pages/dynamic-portal.tsx

```diff
@@ -523,21 +523,14 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   }
 
   const prisma = prismaClient;
-  const environmentToken = process.env.DYNAMIC_TEMPLATES_ENVIRONMENT_ID;
-  const userId = token.sub;
 
+  const environmentToken = process.env.DYNAMIC_TEMPLATES_ENVIRONMENT_ID;
   if (!environmentToken) {
     console.error("Missing DYNAMIC_TEMPLATES_ENVIRONMENT_ID env var");
     throw "Missing DYNAMIC_TEMPLATES_ENVIRONMENT_ID env var";
   }
-  const space = await createSpace({
-    workflow: WorkflowType.Dynamic,
-    userId,
-    environmentId: environmentToken,
-    spaceName: "HCM.show Dynamic",
-  });
```

Comment:
Don't want to automatically create a space, only if we don't have one for this user. This could speed up the workbook lookup if they load the page again